src/Client/appsettings.json

```diff
@@ -25,37 +22,19 @@
         "url": "https://docs.steeltoe.io/articles/releases/steeltoe-3-1-minor-release-major-deal.html"
       },
       {
-        "date": "<span>June</span> 14 2021",
-        "title": "Steeltoe 3.1.0 Release Candidate 2 is now available",
-        "description": "This is the second and final release candidate for Steeltoe 3.1.0. Take a look and tell us what you think.",
-        "url": "https://github.com/SteeltoeOSS/Steeltoe/releases/tag/3.1.0-rc2"
-      },
-      {
-        "date": "<span>May</span> 10 2021",
-        "title": "Steeltoe 3.1.0 Release Candidate 1 is now available",
-        "description": "Steeltoe 3.1.0 brings the new Steeltoe Stream feature built on our RabbitMQ messaging components to the .NET community. Take a look and give us your thoughts.",
-        "url": "https://github.com/SteeltoeOSS/Steeltoe/releases/tag/3.1.0-rc1"
-      },
-      {
-        "date": "<span>january</span> 14 2021",
+        "date": "<span>January</span> 14 2021",
```

Comment:
Do these calendar events show up anywhere on the website? (Other than a banner for the current one, based on the current time)

Reply:
These show up on the home page